src/k8s/pkg/component/component_test.go

```diff
@@ -200,3 +216,68 @@ func TestListComponentsWithReleases(t *testing.T) {
 		{Name: "two", Status: true},
 	}))
 }
+
+func TestComponentsInitialState(t *testing.T) {
+	g := NewWithT(t)
+
+	mockHelmClient, _, _ := mustCreateNewHelmClient(t, map[string]types.Component{
+		"one": {ReleaseName: "whiskas-1", Namespace: "default", ManifestPath: "chunky-tuna-0.1.0.tgz"},
```

Comment:
Miny tweak: Can we either re-format my releases to match your formatting or vice versa? Does `make go.fmt` take care of this?